skydive-grafana/docker/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM debian:jessie
+
+MAINTAINER Marcos Entenza (mak@redhat.com)
+
+ENV GID 28000
+ENV UID 28000
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN groupadd --gid $GID grafana && useradd --uid $UID -m -g grafana grafana
```

Comment:
could we find a way around creating guids/uids in a container? This won't pass normal good practice scc policy in openshift and I wouldn't want to recommend stuff that goes against our security recommendations